lib/Search/SearchTablesProvider.php

```diff
@@ -96,17 +96,21 @@ public function search(IUser $user, ISearchQuery $query): SearchResult {
 		$offset = $query->getCursor();
 		$offset = $offset ? (int) $offset : 0;
 
+		// TODO how to read if dark mode is set to use light icons?
```

Comment:
There doesn't seem to be a proper way to do this as of right now.

The only working one is to use a css class as the icon parameter but this would require to load a stylesheet on every page. https://github.com/nextcloud/server/blob/4748626aa4a06a2c5f7d8e3c0bb0b1ddfd4e710e/lib/public/Search/SearchResultEntry.php#L104